Tasks/SonarQubePreBuild/SonarQubePreBuildImpl.ps1

```diff
@@ -1,4 +1,27 @@
 #
+# Orchestrates the pre build task logic
+#
+function InvokePreBuildTask
+{
+    $serviceEndpoint = GetEndpointData $connectedServiceName
+    Write-Verbose "serverUrl = $($serviceEndpoint.Url)"
+
+    $currentDir = (Get-Item -Path ".\" -Verbose).FullName
+    $bootstrapperDir = [System.IO.Path]::Combine($currentDir, "MSBuild.SonarQube.Runner-1.1") # the MSBuild.SonarQube.Runner is version specific
+    $bootstrapperPath = [System.IO.Path]::Combine($bootstrapperDir, "MSBuild.SonarQube.Runner.exe")
+
+    StoreParametersInTaskContext $serviceEndpoint.Url $bootstrapperPath "$($serviceEndpoint.Url)/dashboard/index?id=$($projectKey)" $breakBuild
+    StoreSensitiveParametersInTaskContext $serviceEndpoint.Authorization.Parameters.UserName $serviceEndpoint.Authorization.Parameters.Password $dbUsername $dbPassword
+
+    $cmdLineArgs = UpdateArgsForPullRequestAnalysis $cmdLineArgs $serviceEndpoint
+    Write-Verbose -Verbose $cmdLineArgs
+
+    $arguments = CreateCommandLineArgs $projectKey $projectName $projectVersion $serviceEndpoint.Url $serviceEndpoint.Authorization.Parameters.UserName $serviceEndpoint.Authorization.Parameters.Password $dbUrl $dbUsername $dbPassword $cmdLineArgs $configFile
+
+    Invoke-BatchScript $bootstrapperPath -Arguments $arguments
+}
+
+#
```

Comment:
It doesn't look like any of this code has changed. However, it's not that easy to tell in the web based UI tools. Please help out reviewers by calling out code that just been moved (and if it's been moved and changed then call out the changes).